jpa/hibernate4_3/pom.xml

```diff
@@ -0,0 +1,149 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!--
+  ~  JBoss, Home of Professional Open Source
+  ~  Copyright 2013, Red Hat Middleware LLC, and individual contributors
+  ~  by the @authors tag. See the copyright.txt in the distribution for a
+  ~  full listing of individual contributors.
+  ~
+  ~  Licensed under the Apache License, Version 2.0 (the "License");
+  ~  you may not use this file except in compliance with the License.
+  ~  You may obtain a copy of the License at
+  ~  http://www.apache.org/licenses/LICENSE-2.0
+  ~  Unless required by applicable law or agreed to in writing, software
+  ~  distributed under the License is distributed on an "AS IS" BASIS,
+  ~  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~  See the License for the specific language governing permissions and
+  ~  limitations under the License.
+  -->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>org.wildfly</groupId>
+        <artifactId>wildfly-jpa-parent</artifactId>
+        <version>10.0.0.Alpha2-SNAPSHOT</version>
+    </parent>
+
+    <properties>
+        <version.org.hibernate4_3>4.3.5.Final</version.org.hibernate4_3>
+        <version.org.hibernate4_3.commons.annotations>4.0.1.Final</version.org.hibernate4_3.commons.annotations>
```

Comment:
true, I removed the version.org.hibernate4_3.commons.annotations property as the wildfly-parent contains the version that we need.